dde-file-manager-lib/views/computerview.cpp

```diff
@@ -274,7 +274,7 @@ bool ComputerViewItem::eventFilter(QObject *obj, QEvent *event)
 bool ComputerViewItem::event(QEvent *event)
 {
     if (event->type() == QEvent::Resize) {
-        resize(width(), getIconLabel()->height() + getTextEdit()->height() + ICON_MODE_ICON_SPACING + 45);
+        resize(width(), getIconLabel()->height() + TEXT_LINE_HEIGHT + ICON_MODE_ICON_SPACING + 90);
```

Comment:
I cannot accept this change since [Calling resize() inside QEvent::Resize can lead to infinite recursion.](https://doc.qt.io/qt-5/qwidget.html#size-prop), so actually this part of code should be rewrite and removed.